.github/workflows/ci.yml

```diff
@@ -27,6 +27,9 @@ jobs:
       - name: install LLVM on Windows
         if: matrix.os == 'windows-latest'
         run: choco install llvm -y
+      - name: Environment
```

Comment:
We should use the `env`  syntax since this value is not determined programmatically: 
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env

Reply:
How do you see it now with https://github.com/ZcashFoundation/zebra/pull/947/commits/7f6c378fcede5cf1a8cb98afb03e2ddf13282a98 ?